plugins/catalog/src/plugin.ts

```diff
@@ -30,7 +30,9 @@ import {
   discoveryApiRef,
```

Comment:
There would be no changes in this file for this PR, it's just an example to try it out.